web-front-end/angular/Dockerfile

```diff
@@ -17,4 +17,4 @@ RUN su node -c "source /usr/local/share/nvm/nvm.sh && nvm install ${EXTRA_NODE_V
 # [Optional] Uncomment if you want to install more global node modules
 # RUN su node -c "npm install -g <your-package-list-here>"
 RUN npm install --ignore-scripts puppeteer
-ENTRYPOINT npm run start-prod
\ No newline at end of file
+ENTRYPOINT npm run ${WEB_FRONT_END_START_MODE:-start-prod}
```

Comment:
@DovOps  should i revert this change?

Reply:
Yeah, if ingress is working you don't need to connect to each service via its port, but rather connect only via the ingress controller, but test to make sure the app works fine that way.

Reply:
Done - also tested on a windows env :)